Pod/Classes/NotificationConvenience.swift

```diff
@@ -11,16 +11,14 @@ import Foundation
 
 public extension NotificationCenter {
     
-    public func post(notification: Notification, onMainQueue mainQueue: Bool) -> Void {
-        func postNote() {
-            post(notification)
+    public func post(notification: Notification, queue: DispatchQueue = DispatchQueue.main) -> Void {
+        queue.async { [unowned self] in
```

Comment:
Is `[unowned self]` correct? Sure, the mainQueue will live the lifetime of the app, but what happens if I create a dispatchQueue and pass it in here, then get rid of it before the block gets called?